packages/comet-uswds/src/components/range-slider/range-slider.tsx

```diff
@@ -42,17 +64,26 @@ export const RangeSlider = ({
   ...inputProps
 }: RangeSliderProps & Omit<JSX.IntrinsicElements['input'], 'min' | 'max'>): React.ReactElement => {
   return (
-    <input
-      id={id}
-      className="usa-range"
-      type="range"
-      min={minValue}
-      max={maxValue}
-      step={step}
-      defaultValue={defaultValue}
-      onChange={onChange}
-      role="slider"
-      {...inputProps}
+    <FormGroup
+      id={`form-group-${id}`}
+      required={required}
+      label={label}
+      helperText={helperText}
+      errors={errors}
+      fieldControl={
+        <input
+          id={id}
+          className="usa-range"
+          type="range"
+          min={minValue}
+          max={maxValue}
+          step={step}
+          defaultValue={defaultValue}
+          onChange={onChange}
+          role="slider"
+          {...inputProps}
+        />
+      }
     />
```

Comment:
What if there were an option to display the minimum and max range on the slider, then maybe the output of where you are at on the range, a number that always floated above the slider circle?

Reply:
I think is is something that could be implemented on an as needed basis with a combination of components. If did want to add that, we could handle through another PR though.